src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -198,17 +196,16 @@ public static enum PositionType {
 	BooleanPublisher armManualOverridePublisher;
 
 	StringPublisher wristGoalPublisher;
-	StringPublisher wristPositionPublisher;
 
 	// Constructor
 
 	public ArmSubsystem() {
-		armMotor1 = new CANSparkMax(ARM_MOTOR1, MotorType.kBrushless);
-		armMotor2 = new CANSparkMax(ARM_MOTOR2, MotorType.kBrushless);
+		armMotor1 = new CANSparkMax(ARM_MOTOR_1, MotorType.kBrushless);
+		armMotor2 = new CANSparkMax(ARM_MOTOR_2, MotorType.kBrushless);
 		wristMotor = new CANSparkMax(WRIST_MOTOR, MotorType.kBrushless);
 
-		shoulderEncoder = new DutyCycleEncoder(SHOULDER_ENCODER_PORT_A);
-		shoulderEncoder.reset();
+		elbowEncoder = new DutyCycleEncoder(ELBOW_ENCODER_PORT);
+		elbowEncoder.reset();
```

Comment:
Mechanical is wiring the elbow encoder to a spark max (like how the wrist encoder is currently set up) not the roboRIO, so this will have to change unfortunately.

I don't think we'll be using the encoder during Glacier Peak though, so could be fine to leave as-is. We'll see if anything complains about the encoder not being plugged in.

Reply:
do i have to use sparkmaxpidcontroller then ?

Reply:
Yeah, or use ProfiledPIDController but read the encoder values from the spark max (although that's a bit strange). 

One issue is that sparkmaxpidcontroller doesn't have the same max velocity and max acceleration limiting feature that ProfiledPIDController does. There's smart motion, but it has some problems as discussed here: https://www.chiefdelphi.com/t/understanding-and-tuning-smart-motion-for-an-arm/426639. I'd also recommend reading through that thread to learn about the different options we have. 

All this might be a bit much for arm presets we aren't planning on using during Glacier Peak, so feel free to resolve, merge, and start working on this after this weekend.

Reply:
Lemme add some method descriptions then i'll merge